Apromore-Core-Components/Apromore-Manager/src/test/java/org/apromore/service/impl/WorkspaceServiceImplTest.java

```diff
@@ -362,5 +353,122 @@ public void testMoveFolder() throws Exception {
         Assert.assertEquals(folder.getName(), movedFolder.getName());
         Assert.assertEquals(newParentFolder, movedFolder.getParentFolder());
     }
-    
+
+    @Test
+    public void getLogACL() {
+
+        // Set up test data
+        Group group1 = createGroup(1, Group.Type.GROUP);
+        Group group2 = createGroup(2, Group.Type.GROUP);
+        Group group3 = createGroup(3, Group.Type.USER);
+        Group group4 = createGroup(4, Group.Type.USER);
+
+        Role role = createRole(createSet(createPermission()));
+        User user = createUser("userName1", group1, createSet(group1), createSet(role));
+
+        Workspace wp = createWorkspace(user);
+        Log log = createLog(user, createFolder("testFolder", null, wp));
+
```

Comment:
Unit test methods should be named as testXXX, e.g. testGetLogACL, or test_getLogACL, testGetProcessACL, etc.